external/criu/pingPerf.sh

```diff
@@ -122,19 +122,22 @@ findCommandAndReplace() {
     local oldCmd="$1"
     local newCmd="$2"
     local fileName="$3"
+    local mustFind="$4"
     # Default sed delimiter is ":"
     local sedDelimiter=":"
-    if [ ! -z "$4" ]; then
-        sedDelimiter="$4"
+    if [ ! -z "$5" ]; then
+        sedDelimiter="$5"
     fi
 
     echo "start grep: grep -c \"$oldCmd\" $fileName"
     local occurrences=$(grep -c "$oldCmd" $fileName)
-    if [[ $occurrences -eq 0 ]]; then
+    if [[ $occurrences -eq 0 && "$mustFind" == "true" ]]; then
         echo "Error: The command '$oldCmd' was not found in $fileName."
         exit 1
+    elif [[ $occurrences -eq 0 && "$mustFind" == "false" ]]; then
+        echo "No occurrence of $oldCmd, ignore and continue."
```

Comment:
Can we do the following?
```
if [ $occurrences -eq 0 ]; then
   if [ "$mustFind" == "true" ]; then
       echo "Error: The command '$oldCmd' was not found in $fileName."
       exit 1
   else
       echo "No occurrence of $oldCmd, ignore and continue."
    fi
else
...

```

Reply:
Updated. New test link is hyc_grinder 46399. thanks.